### DISCUSSION TITLE
RNode battery

### DISCUSSION THREAD

**Discussion by aj5r1gg** [markqvist/Reticulum#389] [Help & Questions]
Firstly, I'd like to thank Mark for sending my brand new RNode device. I live in Australia and it arrived faster from Europe than some local deliveries. Woohoo!  (After four weeks, I'm still waiting on a delivery from a supplier that's three hours drive away).

Secondly, I am having trouble sourcing a battery in Australia for the device. 

From Mark, I ordered the Handheld RNode with enough space to handle a 1100mAh battery (a battery that is no larger than 50mm x 25mm x 12mm). No issues or dramas there.

However, the LilyGO LoRa32 v2.1 board specifies a connector for the battery (power supply) as JST GH 2pin 1.25mm. 

I have tried electronics specialists and battery specialists in Australia for such JST connectors to no avail. I have contacted the LilyGO manufacturer and they've only given a very general response as "suggest a 1.25mm connector." 

At present, I am left with ordering such connectors over eBay or Amazon. 

Is there an alternative? Is there another way? What am I missing?

**Comment by faragher** [comment]
I have yet to find a good source. I was forced to clip the old connector and attach a new one (note, halfway down the wires, don't solder the battery itself) which was honestly one of the more stressful things I've done this year. If you decide to do so, keep a metal pan around in case it shorts. 

I used the connector that comes with the LilyGO boards, I'm not sure if it's included with the RNode. 

If you find a stock solution, please let me know. I don't mind doing this to a handful of batteries, but eventually scale will kill me.

**Comment by markqvist** [comment]
Huh, that's odd. For me they have always been easy to find, but maybe the situation is different in Australia. The 1.25mm JST connector is also called "Micro-JST", maybe that will turn up more results when searching. Either way, something like this works well, and has the correct connector already attached:

https://www.amazon.de/-/en/Lithium-Protective-Insulation-Connector-Development/dp/B087LTZW61/

If it's impossible to find something suitable locally, I'd be happy to throw a few Micro-JST connector pigtails in an envelope and send you, that you can use to solder to whatever battery you can actually get locally.

If you do it like that, just do the wires one at a time, and it's easy, with no risk of shorting. Snip the ground wire first, then solder on the ground lead of the connector (remember some shrink-wrap insulation). With the ground lead soldered and re-insulated, snip the positive wire, solder and apply insulation, and you're done.

Maybe I should actually include one of those connector leads with every RNode by default, now that I think of it.

**Comment by aj5r1gg** [comment]
Thank you @markqvist and @faragher for your responses. I haven't much of an alternative. I'll just have to order the 1.5mm JST connectors over Amazon.
